spec/index/index_spec.rb

```diff
@@ -102,6 +120,11 @@
       subject { Daru::Index.new ('a'..'z').to_a  }
       its(:inspect) { is_expected.to eq "#<Daru::Index(26): {a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t ... z}>" }
     end
+
+    context 'index with name' do
+      subject { Daru::Index.new ['one', 'two', 'three'], name: 'number'  }
+      its(:inspect) { is_expected.to eq "#<Daru::Index: number(3): {one, two, three}>" }
```

Comment:
I am not sure it is readable (two colons and stuff). For larger classes (with multiline inspect) we typically have name after `#<ClassName(size)`, let make it the same way: `"#<Daru::Index(3): number {one, two, three}>"`. Here `Daru::Index(3)` is "system" part, and everything else is "content".
@v0dro, WDYT?